apps/speed.c

```diff
@@ -3581,6 +3583,21 @@ static int do_multi(int multi, int size_num)
         fclose(f);
     }
     OPENSSL_free(fds);
+    for (n = 0; n < multi; ++n) {
+        while (wait(&status) == -1) {
+            if (errno != EINTR) {
+                BIO_printf(bio_err, "Waitng for child failed with 0x%x\n",
+                           errno);
+                return 1;
+            }
+        }
```

Comment:
We'd normally drop this pair of `{}` since it's unnecessary.

Reply:
The one from the `while`?  Shall I drop it?

Reply:
Yes, and yes.

Reply:
Done.